<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Fx"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>TORQUE X:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_Tx"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>FORCE Y:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>FORCE X:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_SpeedJ">
      <property name="text">
       <string>SpeedJ</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Fy"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Fz"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>TORQUE Z:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>TORQUE Y:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit_Ty"/>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButton_ForceMode">
      <property name="text">
       <string>ForceMode</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="lineEdit_Tz"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>FORCE Z:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QLineEdit" name="lineEdit_Connection">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButton_Home">
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_MoveJ">
        <property name="text">
         <string>MoveJ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_moveJ">
        <property name="text">
         <string>Tracking</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_Servoc">
        <property name="text">
         <string>Servoc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Servoc">
        <property name="text">
         <string>Tracking</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuustawienia">
    <property name="title">
     <string>Ustawienia</string>
    </property>
    <addaction name="actionUstawienia_okna"/>
    <addaction name="actionConnection"/>
    <addaction name="actionDisconnection"/>
   </widget>
   <widget class="QMenu" name="menuWayPoint">
    <property name="title">
     <string>WayPoint</string>
    </property>
    <addaction name="actionParameters"/>
    <addaction name="actionPlane_Callibration"/>
   </widget>
   <widget class="QMenu" name="menuRejestracja">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Rejestracja</string>
    </property>
    <addaction name="actionRozpocznij"/>
    <addaction name="actionZakoncz"/>
    <addaction name="actionPlik"/>
   </widget>
   <addaction name="menuustawienia"/>
   <addaction name="menuWayPoint"/>
   <addaction name="menuRejestracja"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUstawienia_okna">
   <property name="text">
    <string>Ustawienia okna</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionConnection">
   <property name="text">
    <string>Connection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="text">
    <string>Parameters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionRozpocznij">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rozpocznij</string>
   </property>
  </action>
  <action name="actionZakoncz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zako≈Ñcz</string>
   </property>
  </action>
  <action name="actionPlik">
   <property name="text">
    <string>Plik</string>
   </property>
  </action>
  <action name="actionDisconnection">
   <property name="text">
    <string>Disconnection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPlane_Callibration">
   <property name="text">
    <string>Plane callibration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
